.env

```diff
@@ -10,7 +10,7 @@
 ##
 
 # The site will be available at http://localhost:LOCAL_PORT
-LOCAL_PORT=8889
+LOCAL_PORT=8890
```

Comment:
Is this intentional? If so, can you just give the reasoning for the change?

Reply:
Oh whoops, that shouldn't have been committed! I wish this file could be gitignored as it's easier to change settings here than set custom values in terminal every time 😅

Reply:
No worries! I figured it was accidental, but just wanted to make sure 😄